sibr-cassandra-migrations.cql

```diff
@@ -103,13 +103,45 @@ CREATE TABLE IF NOT EXISTS blaseball.games(
   perceived_ended_at timestamp
 );
 
-CREATE TABLE IF NOT EXISTS blaseball.game_events_numeric(
+CREATE TYPE blaseball.event_base_runner(
+  runner_id uuid,
+  responsible_pitcher_id uuid,
+  base_before_play tinyInt,
+  base_after_play tinyInt,
+  was_base_stolen boolean,
+  was_caught_stealing boolean,
+  was_picked_off boolean,
+);
+
+CREATE TABLE IF NOT EXISTS blaseball.game_events(
   perceived_at timestamp,
   game_id uuid,
   event_type text,
-  player_id uuid,
-  inning smallInt,
-  numeric_value decimal,
-  string_value text,
-  PRIMARY KEY (game_id, perceived_at)
+  event_index int,
+  inning tinyInt,
+  outs_before_play tinyInt,
```

Comment:
Do we need balls and strikes count as well?

Reply:
yes I knew I forgot something lol

Reply:
The retrosheet list is significantly longer but many of the field names are rather obscure, can't tell quickly if anything else important is missing.

Reply:
There's a lot of fielder position fields that can't track.